Sample log from the pod when the error was noticed:

```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib64/python2.7/SimpleHTTPServer.py", line 220, in <module>
    test()
  File "/usr/lib64/python2.7/SimpleHTTPServer.py", line 216, in test
    BaseHTTPServer.test(HandlerClass, ServerClass)
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 595, in test
    httpd = ServerClass(server_address, HandlerClass)
  File "/usr/lib64/python2.7/SocketServer.py", line 419, in __init__
    self.server_bind()
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/usr/lib64/python2.7/SocketServer.py", line 430, in server_bind
    self.socket.bind(self.server_address)
  File "/usr/lib64/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 13 Permission denied
```

--- a/.github/workflows/e2e-full.yml
+++ b/.github/workflows/e2e-full.yml
@@ -46,6 +46,7 @@ jobs:
         with:
           using: ${{ matrix.external_net }} ${{ matrix.globalnet }}
           skip: ""  # Override skipping external network tests
+          plugin: "scripts/e2e/external/hook"
 
       - name: Post mortem
         if: failure()

--- a/scripts/e2e/external/utils
+++ b/scripts/e2e/external/utils
@@ -65,7 +65,7 @@ function setup_external() {
 
     # Create external test container on pseudo-ext network
     docker run --cap-add=NET_ADMIN --name "${EXTERNAL_APP}" --network "${EXTERNAL_NET}" \
-        -d registry.access.redhat.com/ubi7/ubi python -m SimpleHTTPServer 80
+        -d registry.access.redhat.com/ubi7/ubi python -m SimpleHTTPServer 1234
 
     # Add extra routing in external app
     docker exec -i "${EXTERNAL_APP}" yum install -y iproute

--- a/test/external/dataplane/connectivity.go
+++ b/test/external/dataplane/connectivity.go
@@ -21,6 +21,7 @@ package dataplane
 import (
 	"fmt"
 	"sort"
+	"strconv"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
@@ -36,7 +37,7 @@ const (
 	testContainerName = "ext-test-container"
 )
 
-var simpleHTTPServerCommand = []string{"python", "-m", "SimpleHTTPServer", "80"}
+var simpleHTTPServerCommand = []string{"python", "-m", "SimpleHTTPServer", strconv.Itoa(framework.TestPort)}
 
 type testParams struct {
 	Framework         *framework.Framework
@@ -195,7 +196,7 @@ func testExternalConnectivity(p testParams) {
 
 	np := p.Framework.NewNetworkPod(&framework.NetworkPodConfig{
 		Type:          framework.CustomPod,
-		Port:          80,
+		Port:          framework.TestPort,
 		Cluster:       p.Cluster,
 		Scheduling:    p.ClusterScheduling,
 		Networking:    p.Networking,
@@ -229,7 +230,7 @@ func testExternalConnectivity(p testParams) {
 	By(fmt.Sprintf("Sending an http request from external app %q to %q in the cluster %q",
 		dockerIP, targetIP, clusterName))
 
-	command := []string{"curl", "-m", "3", fmt.Sprintf("%s:%d/%s%s", targetIP, 80, p.Framework.Namespace, clusterName)}
+	command := []string{"curl", "-m", "3", fmt.Sprintf("%s:%d/%s%s", targetIP, framework.TestPort, p.Framework.Namespace, clusterName)}
 	_, _ = docker.RunCommandUntil(command...)
 
 	By("Verifying the pod received the request")
@@ -245,7 +246,7 @@ func testExternalConnectivity(p testParams) {
 	By(fmt.Sprintf("Sending an http request from the test pod %q %q in cluster %q to the external app %q",
 		np.Pod.Name, podIP, clusterName, dockerIP))
 
-	cmd := []string{"curl", "-m", "10", fmt.Sprintf("%s:%d/%s%s", dockerIP, 80, p.Framework.Namespace, clusterName)}
+	cmd := []string{"curl", "-m", "10", fmt.Sprintf("%s:%d/%s%s", dockerIP, framework.TestPort, p.Framework.Namespace, clusterName)}
 	_, _ = np.RunCommand(cmd)
 
 	By("Verifying that external app received request")

